nemo_skills/inference/llm_math_judge.py

```diff
@@ -109,10 +115,12 @@ def generate(cfg: LlmMathJudgeConfig):
     task.generate()
 
 
-HELP_MESSAGE = get_help_message(
-    LlmMathJudgeConfig,
-    server_params=server_params(),
-    sandbox_params=sandbox_params(),
+HELP_MESSAGE = (
+    LlmMathJudgeConfig.__doc__ + "\n\n" + 
```

Comment:
Should we do this inside get_help_message, so that it automatically does it for any class?

Reply:
Done!